src/config/configurations.ts

```diff
@@ -0,0 +1,29 @@
+import {Configuration} from './configuration';
+import {
+  ITransportStrategy,
+  StaticGrpcConfiguration,
+  StaticTransportStrategy,
+} from './transport/transport-strategy';
+import {IGrpcConfiguration} from './transport/grpc-configuration';
+
+export class Laptop extends Configuration {
+  constructor(transportStrategy: ITransportStrategy, maxIdleMillis: number) {
+    super(transportStrategy, maxIdleMillis);
+  }
```

Comment:
Redundant, it's already inherited.

Reply:
are you saying that in JS/TS a child class will inherit non-default constructors from the parent class?  If so, that seems surprising to me!  I don't know if I've seen that in a language before.

Even if that's the case I might still be tempted to keep this here just so that it's really clear and explicit when you're looking at this file, but I will play with this in my follow-up PR.

Reply:
Inheriting a constructor seems normal to me, I'd be surprised if it didn't.

> Classes may extend from a base class. A derived class has all the properties and methods of its base class, and also define additional members.

Tests pass if you delete it.